walk-in-interview/src/main/webapp/common-functions.js

```diff
@@ -60,5 +60,26 @@ function renderSelectOptions(select, options) {
   }
 }
 
+/**
+ * Gets the value of a specified cookie.
```

Comment:
nit: include why we need this cookie function

Reply:
![Screenshot 2020-08-03 at 5 01 08 PM](https://user-images.githubusercontent.com/49228945/89169195-7dd08780-d5b0-11ea-82c0-7dc614378c0d.png)

Reply:
I think you should include something about this in the docs for the function e.g., this function is needed to help get the idToken for firebase auth ...

Reply:
I don't think so, since `getCookie` is a very commonly used function (that is also why i put it inside common-functions.js), just in this case, it is used only once